### DISCUSSION TITLE
page.curves

### DISCUSSION THREAD

**Discussion by valentinavalenzi** [jsvine/pdfplumber#1080] [Ask for help with specific PDFs]
Hello! Hope you are doing very well. I was hoping if there´s any way I can help the method page.curves . This is because I am trying to get a specific curve in a PDF and it is taking everything around it but that one. Maybe it´s because its stroke isn't very visible. So I heightened its resolution and cropped the image in order for it to only see that curve but still did'nt work. Is there any way in which I can see how that method works and how it decides to take a curve? So that I can catch the one that I need.

**Comment by valentinavalenzi** [comment]
I am asking this because in the example in the documentation, the document does not have the best resolution but still it cateches every curve with no problem.
![image](https://github.com/jsvine/pdfplumber/assets/92260481/c01daca1-632c-49c6-a2c7-89b1597a6bdc)
In contrast, I have an image which I have cropped and added contrast to it but still it doesn't catch a thing whenever I use page.curves
![image](https://github.com/jsvine/pdfplumber/assets/92260481/a0e4a2b8-4007-4d9c-ba9e-51d1c52fb3c5)
That´s my image. I also tried catching the gridlines with the code from the example and I cannot do that either.

**Reply by jsvine** [reply]
Hi @valentinavalenzi,  interesting question. Without having access to the PDF itself, it's hard to say for sure, but my best guess from the screenshot is that curve you're trying to capture is represented in the PDF not as a set of points, but rather as an image.

(For general reference: Stroke width should not have any affect on the accessibility of curve objects in `pdfplumber`.)
